compiler/onecc-docker/README.md

```diff
@@ -0,0 +1,263 @@
+# onecc-docker
+
+`onecc-docker` supports `one-cmd`, which is currently only supported by ubuntu 18.04, in more OS.
+
+
+
+## Description
+
+For now, `one-cmds` tools only support Ubuntu 18.04 and 20.04(not officially).
+So people in other environments can't use our tools unless they upgrade the OS (or install Ubuntu OS).
+
+Therefore, we developed `onecc` that runs using a docker so that users can use `one-cmds` more easily. This is `onecc-docker.`
```

Comment:
```suggestion
Therefore, we developed `onecc` that runs using a docker so that users can use `one-cmds` more easily. This is _onecc-docker_.
```

Reply:
> we developed `onecc` that runs using a docker

I understand this as `onecc` is already running in docker

```suggestion
To overcome this limitation, we provide _onecc-docker_ that runs using a docker so that users can use `one-cmds` more widely.
```